app/main.py

```diff
@@ -1,8 +1,34 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center, clean_power, average_rating, count_of_ratings) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self: list, cars: list) -> list:
+        return [car for car in cars if car.clean_mark < self.clean_power]
```

Comment:
After determining which cars can be washed, you should update their `clean_mark` to the `clean_power` of the `CarWashStation`. This is necessary to reflect that the cars have been washed.